proto/buf/alpha/registry/v1alpha1/authz.proto

```diff
@@ -76,6 +76,15 @@ service AuthzService {
   // UserCanSeeServerAdminPanel returns whether the user is authorized
   // to see server admin panel.
   rpc UserCanSeeServerAdminPanel(UserCanSeeServerAdminPanelRequest) returns (UserCanSeeServerAdminPanelResponse);
+  // UserCanManageRepositoryContributor returns whether the user is authorized to manage
+  // any contributors to the repository and the list of roles they can manage.
+  rpc UserCanManageRepositoryContributor(UserCanManageRepositoryContributorRequest) returns (UserCanManageRepositoryContributorResponse);
+  // UserCanManagePluginContributor returns whether the user is authorized to manage
+  // any contributors to the plugin and the list of roles they can manage.
+  rpc UserCanManagePluginContributor(UserCanManagePluginContributorRequest) returns (UserCanManagePluginContributorResponse);
+  // UserCanManageTemplateContributor returns whether the user is authorized to manage
+  // any contributors to the template and the list of roles they can manage.
+  rpc UserCanManageTemplateContributor(UserCanManageTemplateContributorRequest) returns (UserCanManageTemplateContributorResponse);
```

Comment:
from the previous experience of organization roles, it seems like we don't actually need to separate the roles that user can add/update/remove, they should be the same. If this works, we might want to also update the organization role authz